extra/ios-websocket-hack.js

```diff
@@ -17,6 +17,9 @@
 
 
 (function () {
+    // run on iOS Cordova only
+    if (!(window.cordova && window.cordova.platformId === 'ios')) return;
+
 	// Store a reference of the native WebSocket class.
 	var NativeWebSocket = window.WebSocket;
 
```

Comment:
May you please keep the project indentation and use TAB instead of spaces? Running `gulp` should throw one or more lint/style errors.